components/TextFields/src/ContainedInputView/MDCBaseTextField.m

```diff
@@ -490,7 +492,7 @@ - (UIFont *)normalFont {
 }
 
 - (UIFont *)floatingFont {
-  return [self.normalFont fontWithSize:(self.normalFont.pointSize * (CGFloat)0.5)];
+  return [self.containerStyle floatingFontWithNormalFont:self.normalFont];
```

Comment:
This seems like it would result in a snapshot delta. If we dont already I would like to make sure that we have one in place.

Reply:
It doesn't result in a snapshot delta because I changed the floating font scale factor. This stuff is still in flux a bit.